README.md

```diff
@@ -162,6 +162,9 @@ Mixed line ending........................................................Passed
 Don't commit common R artifacts......................(no files to check)Skipped
 ```
 
+[![Stargazers repo roster for @insightsengineering/r.pkg.template](https://reporoster.com/stars/insightsengineering/r.pkg.template)](https://github.com/insightsengineering/r.pkg.template/stargazers)
```

Comment:
Add these below "Stargazers over time" and change the title of "Stargazers over time" to "Stargazers and Forkers"

Reply:
done
![image](https://user-images.githubusercontent.com/86738093/187483116-c05c3990-d963-40d4-aeb5-cdfd7a508c58.png)